internal/api/versionsapi/cli/rm.go

```diff
@@ -631,7 +648,7 @@ func (a *azureClient) deleteImage(ctx context.Context, image string, dryrun bool
 	}
 
 	if dryrun {
-		log.Debug(fmt.Sprintf("DryRun: delete image %v", azImage))
+		log.Debug(fmt.Sprintf("DryRun: delete image with gallery %q, image definition %q, resource group %q and version %q", azImage.gallery, azImage.imageDefinition, azImage.resourceGroup, azImage.version))
```

Comment:
```suggestion
		log.Debug(fmt.Sprintf("DryRun: delete image: resource group: %q, gallery: %q, image definition: %q, version: %q"", azImage.resourceGroup, azImage.gallery, azImage.imageDefinition, azImage.version))
```

nit: adhere to the log format of the other message later on in this function